src/build_log.cc

```diff
@@ -45,9 +45,9 @@
 
 namespace {
 
-const char kFileSignature[] = "# ninja log v%d\n";
+const char kLogFileSignature[] = "# ninja log v%d\n";
 const int kOldestSupportedVersion = 4;
-const int kCurrentVersion = 5;
+const int kLogCurrentVersion = 5;
```

Comment:
They are required because otherwise the amalgamation contains multiple conflicting definitions of `kFileSignature` and `kCurrentVersion`.